docs/examples.rst

```diff
@@ -8,7 +8,6 @@ Tutorials
 .. toctree::
    :maxdepth: 1
 
-   what_is_vagga
```

Comment:
Well, I don't think it's a big deal. If you find a better way to place link there, do it. Currently, it's more helpful than distracting.